src/azure-cli/azure/cli/command_modules/acs/custom.py

```diff
@@ -1768,6 +1786,10 @@ def _aks_browse(
                     logger.warning("Currently in a test environment, the proxy is closed due to a preset timeout!")
                     return_msg = return_msg if return_msg else ""
                     return_msg += "Test Passed!"
+                except subprocess.CalledProcessError as new_err:
+                    raise CLIError('Could not open proxy: {} Command output: {}'.format(new_err, new_err.output))
+            else:
+                raise CLIError('Could not open proxy: {} Command output: {}'.format(err, err.output))
```

Comment:
Could you please use the specific error types instead of `CLIError`?

Reply:
Sure, resolved in commit[9b74ec0](https://github.com/Azure/azure-cli/pull/19115/commits/9b74ec07d97b88cedb689a217ab22270428b74ff).